superdesk/default_settings.py

```diff
@@ -446,6 +455,9 @@ def env(variable, fallback_value=None):
 #: The number of minutes before published content items are purged
 PUBLISHED_CONTENT_EXPIRY_MINUTES = int(env('PUBLISHED_CONTENT_EXPIRY_MINUTES', 0))
 
+#: The number of minutes before audit content is purged
+AUDIT_EXPIRY_MINUTES = int(env('AUDIT_EXPIRY_MINUTES', 0))
```

Comment:
In production we have 5 Million records consuming 20Gb after around 6 Months. It is becoming a concern as far as storage is concerned.

It has been quite useful for diagnosing production issues but the size is starting to be a problem for searching as well. 

In terms of a value for the parameter I was thinking along the lines of a couple of months. I'd deliberately defaulted it to zero so that an installation would need to opt in to have the collection purged.